ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-aro-4.12.yaml

```diff
@@ -42,12 +42,14 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aro-f28
-  cron: 26 22 20 * *
+- as: aro-f7
+  cron: 36 3 1,8,17,24 * *
   steps:
     cluster_profile: azure-qe
+    env:
+      ARO_CLUSTER_VERSION: 4.12.25
     test:
-    - chain: openshift-upgrade-qe-test
+    - chain: openshift-e2e-test-qe-upgrade
```

Comment:
I am not sure if it is proper to introduce this new chain for a upgrade job, cc @liangxia for more insight.

Reply:
If you need a new upgrade chain for aro, perhaps name it as `openshift-upgrade-qe-test-aro` ?

Reply:
@jianlinliu  @liangxia  
I was asked by @yufchang to switch from the cucushift chain to the openshift-e2e-test-qe-test chain as it has more useful tests in it at this point. However openshift-e2e-test-qe-test does not do upgrades like the old cucushift one did. Since both chains share the same refs, Prow wouldn't let me run the same step more than once, so I can either break down the openshift-e2e-test-qe-upgrade chain into it's individual ref's and put them all in the job config or create a new chain.  If you would rather me rename it to openshift-e2e-test-qe-upgrade-aro or something similar I can do that instead. Let me know which you prefer.

Reply:
`openshift-e2e-xxx` is for e2e, while `openshift-upgrade-yyy` is for upgrade. If the main purpose is to test upgrade and test e2e additionally, I would prefer create a new chain under `openshift/upgrade/qe/test/`